### PR TITLE
KFSPTS-33161 remove code that updates active from date on existing records

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java [hunk 1]
```diff
@@ -771,8 +771,8 @@ protected List<RoleMember> populateRoleMembers(
                         if (roleMember.getActiveFromDate() != null) {
                             roleMemberImpl.setActiveFromDateValue(
                                     new java.sql.Timestamp(roleMember.getActiveFromDate().getTime()));
-                        } else {
-                            // CU Customization KFSPTS-27162 Set default for activeFromDate field
+                        } else if(!roleMember.isEdit()){
```
Comment (cah292) [inline src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java:774]: Missing whitespace before and after the parenthesized condition.

File: src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java [hunk 2]
```diff
@@ -771,8 +771,8 @@ protected List<RoleMember> populateRoleMembers(
...
+                            // CU Customization If this is a new role and activeFromDate is null, set default for activeFromDate field
                             roleMemberImpl.setActiveFromDateValue(dateTimeService.getCurrentTimestamp());
```
Comment (cah292) [inline src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java:776]: This code appears to be affecting new role members that are not necessarily on a new role. Please update this comment for clarity.

**Review by cah292 — CHANGES_REQUESTED** [review]
There are a couple minor areas that should be adjusted, but otherwise the changes look good.

Just to confirm, is any SQL needed to fix existing records?

File: src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java (hunk 1 above)
Comment (db483) [reply src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java:774]: updated

File: src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java (hunk 2 above)
Comment (db483) [reply src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java:776]: updated

**Review by cah292 — COMMENTED** [review]
Changes look good, will merge once we get the unit tests to pass.

**Comment by cah292** [comment]
retest this please

**Review by cah292 — APPROVED** [review]
Changes look good and unit tests pass. Merging.

**Pull request merged**